src/components/Review/ReviewDecision.tsx

```diff
@@ -1,36 +1,55 @@
 import React from 'react'
-import { Checkbox, Form } from 'semantic-ui-react'
+import { Checkbox, Form, Message } from 'semantic-ui-react'
 import { Decision } from '../../utils/generated/graphql'
 import { DecisionOption } from '../../utils/types'
+import strings from '../../utils/constants'
 
 type ReviewDecisionProps = {
   decisionOptions: DecisionOption[]
   setDecision: (decision: Decision) => void
   isDecisionError: boolean
+  isEditable: boolean
 }
 
 const ReviewDecision: React.FC<ReviewDecisionProps> = ({
   decisionOptions,
   setDecision,
   isDecisionError,
+  isEditable,
 }) => {
+  if (isEditable)
+    return (
+      <Form>
+        {decisionOptions
+          .filter((option) => option.isVisible)
+          .map((option) => (
+            <Form.Field error={isDecisionError} key={option.code}>
+              <Checkbox
+                radio
+                label={option.title}
+                name={option.title}
+                value={option.code}
+                checked={option.value}
+                onChange={() => setDecision(option.code)}
+              />
+            </Form.Field>
+          ))}
+      </Form>
+    )
+  // If review has been submitted/locked:
+  // Check if has one selected option
+  const visibleOptions = decisionOptions.filter((option) => option.isVisible)
```

Comment:
I wonder if we need the three checks here ?

what if we just do `decionOptions.find`, and if null or `no decision` don't display ?

Reply:
Yes that's ok. I was kind of looking for an invalid case, but really unlikely. And in that case it would simply find the first option, which is fine too.